views/albums.hbs

```diff
@@ -0,0 +1,12 @@
+<a href="http://localhost:3000">Back to search</a>
+
+<h1>Albums:</h1>
+<main>
+  {{#each albums}}
+    <div class="block">
+      <h2>{{this.name}}</h2>
+      <img src="{{this.images.0.url}}" alt="Image of {{this.name}}" />
```

Comment:
Image is redundant in alt texts ... most screen readers already announce it anyways ^^